src/app/public/modules/date-range-picker/date-range-picker.component.html

```diff
@@ -0,0 +1,74 @@
+<div *ngIf="isReady"
```

Comment:
Semantic question here... Since this element is assigned the `[formGroup]`, should it actually be a `form` element instead of a `div`?